app/services/search_orders.rb

```diff
@@ -50,6 +39,6 @@ def paginated_results
   end
 
   def using_pagination?
-    params[:per_page]
+    params[:page]
```

Comment:
Was there a specific reason for changing this? I have a vague feeling it's coupled to something specific in the Angular code that calls the related endpoint...

Reply:
When the user was sending only the per_page param the application was breaking, but when only the page params were sent it didn't break.

I think this change can be reverted as the paginations calls to this method seems to been called only with both parameters